manifests/schema.pp

```diff
@@ -49,9 +49,27 @@
   $permissions              = {},
   $tables                   = {},
   $users                    = {},
+  $use_scl                  = $::cassandra::params::use_scl,
```

Comment:
please don't add `::` after `$`, that is a deprecated syntax that was required for Puppet 3.

Reply:
Removed